front/app/containers/EventsShowPage/components/DesktopTopBar.tsx

```diff
@@ -48,6 +56,21 @@ const TopBar = ({ project }: Props) => {
               : clHistory.push(`/projects/${project.attributes.slug}`);
           }}
         />
+        {isAdminUser && event && (
+          <Button
+            buttonStyle="secondary"
+            m="0px"
+            icon="edit"
+            px="8px"
+            py="4px"
+            text={formatMessage(messages.editEvent)}
+            onClick={() => {
+              clHistory.push(
+                `/admin/projects/${project.id}/events/${event.id}`
+              );
```

Comment:
Not related to the image, but I wanted to add an "Edit event" button on the event page directly for admins.

Reply:
Is it just admins that can edit an event or moderators too?

Reply:
Ah good catch - yes moderators too :) I'll add that :+1: Thanks!